cryptoservice/crypto_service.go

```diff
@@ -143,7 +144,17 @@ func CheckRootKeyIsEncrypted(pemBytes []byte) error {
 		return ErrNoValidPrivateKey
 	}
 
-	if !x509.IsEncryptedPEMBlock(block) {
+	if notary.FIPSEnabled() {
```

Comment:
Seems like this could be simplified a bit:

```
if block.Type == "ENCRYPTED PRIVATE KEY" {
    return nil
}

if !notary.FIPSEnabled() && x509.IsEncryptedPEMBlock(block) {
    return nil
}
return ErrRootKeyNotEncrypted
```